apps/researcher/src/app/[locale]/communities/[slug]/[listId]/actions.ts

```diff
@@ -0,0 +1,19 @@
+'use server';
+
+import {objectList} from '@colonial-collections/database';
+import {revalidatePath} from 'next/cache';
+
+interface Props {
+  id: number;
+  revalidatePathName: string;
+}
+
+export async function deleteObjectFromList({id, revalidatePathName}: Props) {
+  await objectList.removeObject(id);
```

Comment:
For consistency, use the same terms? The function uses `delete...` whereas the function from `objectList` uses `remove...`.